aws-wildfly/src/test/java/aws/cloudformation/AwsTestConstants.java

```diff
@@ -5,7 +5,7 @@ public class AwsTestConstants {
     public static final String TAG = "sunstoneKeyTag";
     public static final String NAME_1 = "sunstoneKeyName1";
     public static final String NAME_2 = "sunstoneKeyName2";
-    public static final String instanceName = "${non.existing:eapSunstoneInjectInstance}";
+    public static final String instanceName = "${test.instance.name:eapSunstoneInjectInstance}";
```

Comment:
It may not be clear at the first sight where does `test.instance.name` comes from. I would expect `sunstone.properties`. Can you define it in there? Something like test.instance.name=maven.pom.whatever? This may not be ideal naming, feel free to come up with something more appropriate.